tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/decoders/the-user-shou-id-see-the-message.then.js

```diff
@@ -6,6 +6,7 @@ import {
 } from '../../../pageobjects/wazuh-menu/decoders.page';
 
 Then('The user should see the message', () => {
+  cy.wait(5000);
```

Comment:
at least add a TODO comment to remove them later when we implement a better logic

Reply:
mmm, I don't think adding 'TODO' comments is a good approach, we should remove these functions when we find the best way to do it.